support/util/util.go

```diff
@@ -67,6 +68,13 @@ func DeleteIfNeeded(ctx context.Context, c client.Client, o client.Object) (exis
 	return true, nil
 }
 
+func HostedControlPlaneReleaseImage(hcp *hyperv1.HostedControlPlane) string {
```

Comment:
Same comment than https://github.com/openshift/hypershift/pull/2848/files#r1277401572
also do we need both?

Reply:
One operates on HostedCluster and the other works on HostedControlPlane so I'd say so